protostar-rust/src/pretty_printing.rs

```diff
@@ -68,10 +71,11 @@ fn result_data_to_text(data: &[Felt252]) -> String {
     let mut readable_text = String::new();
 
     for felt in data {
-        let felt_bytes = felt.to_bytes_be();
-        let felt_text = String::from_utf8_lossy(&felt_bytes);
-        readable_text.push_str(&felt_text);
+        readable_text.push_str(&format!("\n    original value: [{felt}]"));
```

Comment:
I cannot `\t` due to `stdout_matches` not recognising 4 spaces as a tab (and I guess we want to keep raw strings in tests)